packages/busybox/busybox.config

```diff
@@ -559,7 +559,7 @@ CONFIG_FEATURE_DEFAULT_PASSWD_ALGO=""
 # Linux Ext2 FS Progs
 #
 CONFIG_CHATTR=y
-# CONFIG_FSCK is not set
+CONFIG_FSCK=y
```

Comment:
Is `fsck` useful on Android?
I'm also assuming it would require root privileges.